### PR TITLE
Initial Implementation of PlayGrid

### PR DESCRIPTION
**Pull request by anupam-git**
This is a very initial implementation of PlayGrid.

Changelog :
  - Added a new PlayGrid page
  - Added a menu entry in main page for PlayGrid
  - Implemented Multi Touchpoints for testing 5-point touch

### CODE REVIEW

File: zynqtgui/zynthian_gui_play_grid.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from . import zynthian_gui_selector
+
+class zynthian_gui_play_grid(zynthian_gui_selector):
```
Comment (notmart) [inline zynqtgui/zynthian_gui_play_grid.py:3]: perhaps should't be a gui_selector page? those are the ones which central ui is a big list of sort so they have a model implementation to do so, this kind opf ui i don't think would have any of that?
would be instead zynthian_qt_gui_base.ZynGui (which yeah, probably needs the case to be synced with the rest :p)

File: qml-ui/pages/PlayGrid.qml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+*/
+
+import QtQuick 2.10
+import QtQuick.Layouts 1.4
+import QtQuick.Controls 2.2 as QQC2
+import org.kde.kirigami 2.4 as Kirigami
+
+import "../components" as ZComponents
+
+ZComponents.SelectorPage {
```
Comment (notmart) [inline qml-ui/pages/PlayGrid.qml:33]: ZComponents.ScreenPage?

Comment (anupam-git) [reply qml-ui/pages/PlayGrid.qml]: Fixed in latest commit

File: zynqtgui/zynthian_gui_play_grid.py (hunk 1 above)
Comment (anupam-git) [reply zynqtgui/zynthian_gui_play_grid.py:3]: Getting quite some errors regarding undefined attributes when inherited from `zynthian_qt_gui_base.ZynGui`

Comment (anupam-git) [reply zynqtgui/zynthian_gui_play_grid.py]: Fixed in latest commit

**Pull request closed without merging**